assets/javascripts/confirm-location/index.js

```diff
@@ -1,9 +1,59 @@
-var map = require('../map')
+var $ = require('jquery')
+var ol = require('openlayers')
+var Map = require('../map')
+var VectorDrag = require('../vector-drag')
 
 function ConfirmLocationPage (options) {
   var easting = window.encodeURIComponent(options.easting)
   var northing = window.encodeURIComponent(options.northing)
-  map.loadMap(easting && [easting, northing])
+  var mapOptions = {
+    point: [parseInt(easting), parseInt(northing)],
```

Comment:
parseint should include 2nd radix arg (usually base 10). Thought standard picked this up.